src/parsers/uniqueTokens.js

```diff
@@ -47,6 +47,8 @@ export const parseAccountUniqueTokens = data => {
         'wiki_link',
       ]),
       familyImage: collection.image_url,
+      familyName:
+        asset_contract.address === ENSAddress ? 'ENS' : collection.name,
```

Comment:
If we add more overrides in the future we should move this into a function but for now it's fine